packages/react-components/src/molecules/AnalyticsControls.tsx

```diff
@@ -150,6 +156,20 @@ const updateSearchParams = (): URLSearchParams => {
   return searchParams;
 };
 
+const getLastUpdated = () => {
+  const lastUpdated = new Date();
+  const currentHour = lastUpdated.getUTCHours();
+
+  if (currentHour <= 6) lastUpdated.setDate(lastUpdated.getDate() - 1);
```

Comment:
Although the sync runs at 6am UTC, since it runs for about an hour,  till 7am UTC the last update would be the previous day's run.

<img width="1057" alt="Screenshot 2024-09-04 at 10 13 34" src="https://github.com/user-attachments/assets/2a194f71-7c10-4d82-b0e0-bd015c105889">